wpilibc/src/main/native/include/frc/simulation/SolenoidSim.h

```diff
@@ -18,9 +18,58 @@ class SolenoidSim {
   SolenoidSim(PneumaticsModuleType type, int channel);
   ~SolenoidSim() = default;
 
+  /**
+   * Returns the solenoid output.
+   * @return the solenoid output.
+   */
+  bool Get() const;
+
+  /**
+   * Returns the solenoid output.
+   * @return the solenoid output.
+   */
+  [[deprecated("Use Get method instead.")]]
   bool GetOutput() const;
```

Comment:
I should've looked at this for longer- Since `IsOn` is equivalent to `GetOutput` we can just go ahead and deprecate `GetOutput` and not add `Get`